src/core/DataFile.cpp

```diff
@@ -898,6 +899,25 @@ void DataFile::upgrade_1_1_91()
 			el.setAttribute( "arpdir", "3" );
 		}
 	}
+
```

Comment:
> `_1_2_0_rc3`

Yes, `1.2.0-rc3` will be the next release assuming the fix is that specific.  If it's only a regression since `1.1.x`, it's preferred to keep it in the existing `1.1.91` section.